README.md

```diff
@@ -1,3 +1,23 @@
+# Overview
+This branch represents the Parallel IO block and loopback VIP in their
+preonboarded state. This README is a step-by-step tutorial of how to onboard
+the PIO block.
+
+The PIO block communicates to the CPU through an AXI4 interface. This interface
+is used to read from and drive the `odata`, `oenable`, and `idata` control
+registers that map to top-level ports. To integrate this block we need make the
+odata, oenable, and idata ports available at the top-level of design and
+connect the AXI4 interface to Rocket's periphery bus.
```

Comment:
I think it would be more correct to call it the "Test Socket's peripheral bus", not "Rocket's". Rocket is just the name of the core, which excludes the interconnect. I could also see calling it the "Rocket core subsystem's peripheral bus", if you think the Test Socket is too confusing of a concept.